### DISCUSSION TITLE
801

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#976] [Comments]
# 801

JVM 컨퍼런스가 있으면 "Java 세상에서 살던 사람이 처음 TS 세계를 만나고 느낀 여러가지 차이점" 을 발표하려고했는데, 시간도 너무 지났고, 발표 준비하기도 쉽지 않은 것 같아 블로그에 시리즈로 시작한다.Java가 구린 언어다를 표현하기 위한 글이기 보다는 전작(다른 언어로 성장하기)과 마찬가지로 타 생태계를 통해 성장할 수 있음을 알리는 글이다자바를 처음 배울때 getter/setter 에 대한 이야기를 많이 들었다.캡슐화 등의 장점을 들으면서 클래스 안에는 항상 무분별하게 getter/setter 를 생성했다.(그때는 Lombok을 배우지 못했던 터라) IDE의 자동 생성 기능을 사용하면서 열심히 getter/setter를 생성했다.물론 예전부터 많은 분들은 무분별한 getter/setter를..

https://jojoldu.tistory.com/801

**Comment by aliwo** [comment]
아우 시원해

**Comment by Octoping925** [comment]
시리즈가 너무 좋아서 2편을 무진장 기다리고 있습니다 😇
